web/build.sh

```diff
@@ -2,6 +2,6 @@
 
 rm -rf dist/
 mkdir dist; 
-sed -e "s|%%FUNCTION_API_URL%%|${FUNCTION_API_URL}|" -e "s|%%PUBLIC_URL%%|${PUBLIC_URL}|" index.html > dist/index.html;
+sed -e "s|%%FUNCTION_API_URL%%|${FUNCTION_API_URL}|" index.html > dist/index.html;
```

Comment:
We shouldn’t need the api url change either really. Can you take a look at removing that as well?

Reply:
sure

Reply:
This will work then because only 1 route can have as`/`.
removing this will break the sample function deployment from `cloud.do.com` however deploying to the DO button and deploying via DOCTL will work.

When users deploy from the `cloud.do` manual intervention is required where the user will have to set routes, env's, output-dir, etc. and if functions are not on the `/` path then it will break.